helix-term/src/commands/typed.rs

```diff
@@ -881,6 +879,9 @@ fn setting(
         _ => anyhow::bail!("Unknown key `{}`.", args[0]),
     }
 
+    cx.editor
+        .config_events
+        .push(tokio_stream::once(ConfigEvent::Update(runtime_config)));
```

Comment:
With the change to put the editor config behind dynamic dispatch, we now have to send updates back up to the application, and let it make the changes there.